alembic/versions/927b7b4bc7bf_fix_learning_progress_table.py

```
@@ -0,0 +1,31 @@
+"""Fix learning_progress table
+
+Revision ID: 927b7b4bc7bf
+Revises: e8d86206ae53
+Create Date: 2025-02-20 01:52:33.055999
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
```

Ruff (0.8.2)

`sqlalchemy` imported but unused

Remove unused import: `sqlalchemy`

(F401)